### ISSUE TITLE
Verbose adds anouncement in Chrome 

### ISSUE THREAD

**Issue by netblue44** [nvaccess/nvda#8403]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
NVDA anounces all the adds that show thru  chrome not those that you have to browse to on a web page.
pressing control nore restarting NVDA solves the problem 
> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.
NVDA should  not anounce adds unless they are intentionally red.
### System configuration:
NVDA version:
all RC, Master fore 2018.2 and next fore  2018.3 
NVDA Installed or portable:
installed 
Other information:
> Example: Running in a VM

Windows version:
Windows 10 Aplril 2018 update.

**Comment by Adriani90** [comment]
@netblue44 I am not able to reproduce this issue with Chrome 74 and NVDA 2019.1.1. Could you please write exactly what you hear and what NVDA should actually say? Thanks.

**Comment by Adriani90** [comment]
We do not have any updates from the initial author since more than 16 months. Closing as abandoned. Moreover, I cannot reproduce it with NVDA 2019.3 Beta 1. Closing also as works for me.
